content/scalability/docs/supporting-infrastructure.md

```diff
@@ -0,0 +1,228 @@
+
+To mount a second volume to your EC2 instances using [eksctl](https://eksctl.io/) you can use a node group with this configuration:
+
+```
+managedNodeGroups:
+  - name: al2-workers
+    amiFamily: AmazonLinux2
+    desiredCapacity: 2
+    volumeSize: 80
+    additionalVolumes:
+      - volumeName: '/dev/sdz'
+        volumeSize: 100
+    preBootstrapCommands:
+      - "systemctl stop containerd"
+      - "mkfs -t ext4 /dev/nvme1n1"
+      - "rm -rf /var/lib/containerd/*"
+      - "mount /dev/nvme1n1 /var/lib/containerd/"
+      - "systemctl start containerd"
+```
+
+If you are using terraform to provision your node groups please see examples in [EKS Blueprints for terraform](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/examples/node-groups/managed-node-groups/main.tf). If you are using Karpenter to provision nodes you can use [`blockDeviceMappings`](https://karpenter.sh/docs/concepts/node-templates/#specblockdevicemappings) with node user-data to add additional volumes.
+
+To mount an EBS volume directly to your pod you should use the [AWS EBS CSI driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver) and consume a volume with a storage class.
+
+```
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ebs-sc
+provisioner: ebs.csi.aws.com
+volumeBindingMode: WaitForFirstConsumer
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ebs-claim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: ebs-sc
+  resources:
+    requests:
+      storage: 4Gi
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: app
+spec:
+  containers:
+  - name: app
+    image: public.ecr.aws/docker/library/nginx
+    volumeMounts:
+    - name: persistent-storage
+      mountPath: /data
+  volumes:
+  - name: persistent-storage
+    persistentVolumeClaim:
+      claimName: ebs-claim
+```
+
+### Avoid instances with low EBS attach limits if workloads use EBS volumes
+
+EBS is one of the easiest ways for workloads to have persistent storage, but it also comes with scalability limitations. Each instance type has a maximum number of [EBS volumes that can be attached](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html). Workloads need to declare what instance types they should run on and limit the number of replicas on a single instance with Kubernetes taints.
+
+EBS volumes are only available within a single Availability Zone (AZ). Workloads that consume a volume will be limited to run within the AZ where the volume is available. If you ‘re using node groups you will need to add AZ specific tolerations to your workload definition after the volume has been created. If you are using Karpenter, nodes will automatically be provisioned in the correct AZ where the volume was created.
```

Comment:
This feel like it belongs in the operational excellence section (to be written)

Reply:
Happy to move it there once we have it :+1: